deepchecks/vision/checks/distribution/image_property_drift.py

```diff
@@ -168,17 +180,87 @@ def compute(self, context: Context) -> CheckResult:
                        f'for the distribution of the following image properties: {properties}.' \
                        '</span>'
 
-            displays = [headnote] + [figures[col] for col in columns_order]
-        else:
-            drifts = None
-            displays = []
+            train_samples = df_train.sample(10)
+            test_samples = df_test.sample(10)
+            train_thumbnail_images, *_ = context.train.batch_of_index(*list(train_samples.index))
+            test_thumbnail_images, *_ = context.test.batch_of_index(*list(train_samples.index))
+
+            thumbnails = self._prepare_thumbnails_block(
+                train_properties=train_samples.T,
+                test_properties=test_samples.T,
+                train_images=train_thumbnail_images,
+                test_images=test_thumbnail_images
+            )
+            displays = [headnote] + [figures[col] for col in columns_order] + [thumbnails]
```

Comment:
are we simply displaying 10 random images from each dataset? Shouldn't we choose the images that represent the drift? (if that is possible, at all)

Reply:
> are we simply displaying 10 random images from each dataset? 

yes

>Shouldn't we choose the images that represent the drift?

yeap, you are right/

> if that is possible, at all

do not know/ 
should it be images from that part of the distribution where the difference is the biggest?